### PR TITLE
Feature/alexander/view candidates of elctions

### PR DESCRIPTION
**Pull request by xand6r**
This PR enables the addition of an interface to view all the candidates in an election.

### CODE REVIEW

File: src/components/pendingElection/components/pendingElection.js [hunk 1]
```diff
@@ -6,6 +6,9 @@ import '../../viewElection/components/ViewElection';
 import { Card, Icon, Spin } from 'antd';
 import { NavLink } from 'react-router-dom';
 import { LOADING_MESSAGE } from '../../viewElection/constants';
+import {
+    VIEW_CANDIDATE_ROUTE, ADD_CANDIDATE_ROUTE, VIEW_CANDIDATE, ADD_CANDIDATE
+} from '../constants';
```
Comment (EngrEric) [inline src/components/pendingElection/components/pendingElection.js:12]: remove the space here

**Review by EngrEric — COMMENTED** [review]
if you are still working on this PR, tag it WIP

**Comment by xand6r** [comment]
[admin should be able to view all the candidates in an election](https://trello.com/c/9S7Sh95Y/67-admin-should-be-able-to-view-all-the-candidates-in-an-election)

**Comment by xand6r** [comment]
[user should be able to see all the candidates in an elections](https://trello.com/c/sjpxrXVC/70-user-should-be-able-to-see-all-the-candidates-in-an-elections)

**Pull request merged**